### PR TITLE
change the job example url k8s.io to kubernetes.io

### PR DESCRIPTION
**Pull request by neo502721**
k8s.io cannot access

### CODE REVIEW

**Comment by gavinfish** [comment]
@neo502721 please sign the CLA first before anyone could start to review this PR.

**Comment by neo502721** [comment]
> @neo502721 please sign the CLA first before anyone could start to review this PR.
I have signed. What's next

File: content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ It takes around 10s to complete.
 You can run the example with this command:
 
 ```shell
-kubectl apply -f https://k8s.io/examples/controllers/job.yaml
```
Comment (gavinfish) [inline content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md]: k8s.io will redirect to kubernetes.io. And if we really want to change this, I suggest to change all the similar links in just one PR.

Comment (neo502721) [reply content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md]: Thank you，but in my cluster the redirection is failure.
[root@node-15 neo]#  kubectl apply -f https://k8s.io/examples/controllers/job.yaml --dry-run        
Unable to connect to the server: dial tcp 35.201.71.162:443: i/o timeout
[root@node-15 neo]# kubectl apply -f https://kubernetes.io/examples/controllers/job.yaml --dry-run

Comment (sftim) [reply content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md]: @neo502721 can you access https://k8s.io/examples/controllers/job.yaml via, eg, `curl`?

(If not, that might not be a website issue)

Comment (neo502721) [reply content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md]: I can't access the url with curl，mabey the reason I‘m in China？

root@node-33:~# curl  https://k8s.io/examples/controllers/job.yaml  -i
curl: (7) Failed to connect to k8s.io port 443: Connection timed out

Comment (sftim) [reply content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md]: https://k8s.io/ is accessible in most of the world. I think Chinese national policy on internet filtering is linked to these symptoms; the Kubernetes infrastructure and website code looks fine.

Comment (sftim) [reply content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md]: Not sure if it's appropriate to /triage unresolved; on the other hand, it doesn't seem like this change should be necessary at all. If this were down to censorship at a corporate firewall I would 100% recommend fixing the firewall.

Comment (kbhawkey) [reply content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md]: Hello @neo502721 . It looks like the docs use both addresses for accessing the example files:
```
https://kubernetes.io/examples/
https://k8s.io/examples/
```
However, I would *not* recommend changing the addresses for the existing example files.
You could always log an issue indicating the problem with accessing the examples.

**Pull request merged**